request.go

```diff
@@ -307,10 +307,10 @@ func (conn *Connection) UpsertAsync(space interface{}, tuple interface{}, ops in
 	})
 }
 
-// CallAsync sends a call to registered Tarantool function and returns Future.
+// Call16Async sends a call to registered Tarantool function and returns Future.
 // It uses request code for Tarantool 1.6, so future's result is always array of arrays
```

Comment:
Maybe we should add the exact version of Tarantool, starting from which an user should use Call instead of Call16 and why?

https://github.com/tarantool/tarantool/blob/fea8c8e40611581d85ab5f96ea5a11e348c02247/src/box/lua/net_box.lua#L760-L765

It is not clear which method should be used in a new code. The user can choose Call16 because it supports more versions of Tarantool.

Reply:
Updated:

```golang
 // Deprecated since 1.7.4. This breaking change was needed to extend call() and eval()
 // API with per-requests options.
 func (conn *Connection) Call16Async(functionName string, args interface{}) *Future {
 	future := conn.newFuture(Call16Request)
```

Reply:
I think just "Deprecated since Tarantool 1.7.4" is enough here (you can add a link to the pull request in Tarantool, but this is not necessary).

The mention of Tarantool's eval() and call() and API can be confusing (an user can think that it's about go-tarantool API).

Reply:
Done.